utils/server_tool.py

```diff
@@ -96,12 +96,22 @@ def send_head(self):
       self.send_header('Content-Length', '0')
       self.end_headers()
       return None
+    if '/delete' in self.path:
+      index = int(self.path.replace('/delete', '').partition('?')[0])
+      csv_list.pop(index)
+      save_to_file()
+      self.send_response(HTTPStatus.MOVED_PERMANENTLY)
+      self.send_header('Location', '/test_set_maker/' + str(index))
+      self.send_header('Cache-Control', 'no-store')
+      self.send_header('Content-Length', '0')
+      self.end_headers()
+      return None
     elif self.path == '/test_set_maker' or self.path == '/test_set_maker/':
```

Comment:
Is this endpoint still being used?

Reply:
Yuh. It's just a nice-to-have so that when the path has no crop id it jumps to the first one.